src/main/java/org/opentripplanner/routing/core/vehicle_sharing/KickScooterDescription.java

```diff
@@ -70,6 +71,11 @@ protected Double getMaximumRangeInMeters() {
         return getDefaultRangeInMeters();
     }
 
+    @JsonIgnore
+    public static double getMaxPossibleSpeed() {
+        return MAX_SPEED_IN_METERS_PER_SECOND_ON_BIKEPATH;
```

Comment:
dlaczego ON_BIKE_PATH jak ma byc szybko to chyba można ulicą?

Reply:
`MAX_SPEED_IN_METERS_PER_SECOND_ON_BIKEPATH` is actually max speed on road that allows bicycles (bike paths and streets). We also use `MAX_SPEED_IN_METERS_PER_SECOND_ON_PEDESTRIAN_PATH` when there is no bike path available.